src/lightning_app/cli/cmd_apps.py

```diff
@@ -0,0 +1,81 @@
+import json
+from datetime import datetime
+
+from lightning_cloud.openapi import Externalv1LightningappInstance, V1LightningappInstanceState
+from rich.console import Console
+from rich.table import Table
+from rich.text import Text
+
+from lightning_app.cli.core import Formatable
+from lightning_app.utilities.cloud import _get_project
+from lightning_app.utilities.network import LightningClient
+
+
+class AppManager:
+    """AppManager implements API calls specific to Lightning AI BYOC apps."""
+
+    def __init__(self):
+        self.api_client = LightningClient()
+
+    def list(self, cluster_id=None):
```

Comment:
It would be nice to have a `output` option, which could be `table|json|yaml|text` for machine readable output.

Reply:
I agree. Let's introduce that in a separate PR though, as it should be honored in all CLI pieces that output text.